src/components/Table.js

```diff
@@ -0,0 +1,168 @@
+import React from "react";
+import PropTypes from "prop-types";
+import styled from "styled-components";
+import grey from "@material-ui/core/colors/grey";
+import ExpandMoreIcon from "@material-ui/icons/ExpandMore";
+
+export default function Table({
+  name,
+  totalRatings,
+  overallRating,
+  ratings,
+  isMore,
+  onLoadMore,
+}) {
+  return (
+    <React.Fragment>
+      <MetaContainer>
+        <div>{name}</div>
+        <div>{`${totalRatings} review${totalRatings >= 1 ? "s" : ""}`}</div>
+        <div>
+          {overallRating > -1
+            ? `Overall ${overallRating}`
+            : "No Overall Rating"}
+        </div>
+      </MetaContainer>
+      <RatingTable>
+        <TableHeaders>
+          <tr>
+            <Header>Information</Header>
+            <Header comment={true}>Comment</Header>
+          </tr>
+        </TableHeaders>
+        <TableBody>
+          {ratings.length > 0 &&
+            ratings.map((rating) => {
+              const date = new Date(rating.rTimestamp);
+              return (
+                <RatingContainer key={rating.id}>
+                  <InfoContainer>
+                    <Rating>{rating.rClass}</Rating>
+                    <Rating>{`Overall: ${rating.rOverall}`}</Rating>
+                    <Rating>{`Helpful: ${rating.rHelpful}`}</Rating>
+                    <Rating>{`${date.getMonth()}/${date.getDate()}/${date.getFullYear()}`}</Rating>
+                  </InfoContainer>
+                  <Comment>{rating.rComments}</Comment>
```

Comment:
There seem to be problems with escaped html. Maybe we can fix this by using the `dangerouslySetInnerHTML` prop.

```suggestion
                  <Comment dangerouslySetInnerHTML={rating.rComments} />
```

Reply:
This break with this error: `Can only set one of `children` or `props.dangerouslySetInnerHTML``

Reply:
This should be resolve with the latest commit. It works! But it might be a security risk for XSS attack though I'm not really sure if that would pose a problem in this case

Reply:
It all really is relying on RMP's parsing which already escapes the characters which could be problematic. I think it should be fine.